src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -426,6 +423,9 @@ FinishRemoteTransactionAbort(MultiConnection *connection)
 		if (!IsResponseOK(result))
 		{
 			const bool isCommit = false;
+
+			HandleRemoteTransactionResultError(connection, result, raiseErrors);
```

Comment:
Thanks for adding this back.

Reply:
I didn't add it, right? It was added in your change, I just put it where it needed to go in the conflict region.